app/src/androidTest/java/net/skyscanner/backpack/docs/DocsRegistry.kt

```diff
@@ -11,6 +11,7 @@ import org.threeten.bp.LocalDate
 object DocsRegistry {
   val screenshots = listOf(
     Screenshot("Badge", "all"),
+    Screenshot("Bar Chart", "default"),
```

Comment:
We should probably have one without legend (default) and one with legend, no?

Reply:
I think we can keep it for now.